source/Generic/PlayState/PlayStateSettingsView.xaml.cs

```diff
@@ -149,5 +154,11 @@ private void SetInformationHotkey(KeyEventArgs e, TextBox tbInformationHotkey, B
             setInformationHotkeyButton.Content = ResourceProvider.GetString("LOCPlayState_SettingChangeHotkeyButtonLabel");
         }
 
+        private bool IsWindows10Or11()
+        {
+            var productName = (string) Microsoft.Win32.Registry.LocalMachine.OpenSubKey(@"Software\Microsoft\Windows NT\CurrentVersion")?.GetValue("productName");
```

Comment:
The key needs to be disposed by using Dispose() method or the `using (...` similar to here https://github.com/darklinkpower/PlayniteExtensionsCollection/blob/24798e311c1db8dfd8eebbd5d1240ad842b20a28/source/Generic/SpecialKHelper/SpecialKHelper.cs#L194-L212 but I will take care of it after merging